be/src/agent/task_worker_pool.cpp

```diff
@@ -1819,11 +1819,11 @@ void* TaskWorkerPool::_report_disk_state_worker_thread_callback(void* arg_this)
             continue;
         }
 #endif
-        vector<RootPathInfo> root_paths_info;
-        worker_pool_this->_env->olap_engine()->get_all_root_path_info(&root_paths_info);
+        vector<DataDirInfo> data_dir_infos;
+        worker_pool_this->_env->olap_engine()->get_all_root_path_info(&data_dir_infos);
```

Comment:
get_all_root_path_info should be renamed?

Reply:
I will modify it.